src/ripple/app/main/Application.cpp

```diff
@@ -844,7 +854,9 @@ class ApplicationImp
 
         mValidations->flush ();
 
-        m_overlay->saveValidatorKeyManifests (getWalletDB ());
+        validators_->stop ();
+
+        manifestCache_->save (getWalletDB ());
```

Comment:
What do you mean keep it around, aren't we already using it? Or does this change bring it back? I'd like not to proliferate databases. In my mind there should be just one database, it simplifies administration. On the other hand, I don't have strong feelings about it so use your best judgement.